microshift_updating/microshift-update-options.adoc

```diff
@@ -63,7 +79,12 @@ You can update {op-system-ostree} or {op-system} without updating {microshift-sh
 
 [id="microshift-update-options-simultaneous-microshift-rhel-updates_{context}"]
 == Simultaneous {microshift-short} and operating system updates
-You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible. Check for compatibility before beginning an update. First use the {op-system-ostree} and {op-system} documentation specific to your update path to plan and update the operating system. Then use the {microshift-short} update type specific to your update path.
+You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible.
+
+* Check for compatibility before beginning an update.
+* First use the {op-system-ostree} and {op-system} documentation specific to your update path to plan and update the operating system.
+* Enable the correct {microshift-short} repository to ensure alignment between your {op-system} and {microshift-short} versions.
+* Then use the {microshift-short} update type specific to your update path.
```

Comment:
Starting a sentence with an adverb might not be everyone's cup of tea. I think the numbering of the steps can remove the "First" and "Then" terms.

![Uploading Screenshot from 2024-08-28 16-50-39.png…]()

Reply:
Yeah, the numerals made those adverbs uneccessary.